config/packages/messenger.yaml

```diff
@@ -1,21 +1,15 @@
 framework:
     messenger:
-        failure_transport: failed
-
         transports:
-
-            async:
+            sms_async:
+                dsn: '%env(MESSENGER_TRANSPORT_DSN)%'
+                options:
+                    queue_name: sms
+            mail_async:
                 dsn: '%env(MESSENGER_TRANSPORT_DSN)%'
                 options:
-                    use_notify: true
-                    check_delayed_interval: 60000
-                retry_strategy:
-                    max_retries: 3
-                    multiplier: 2
-            failed: 'doctrine://default?queue_name=failed'
-            sync: 'sync://'
+                    queue_name: mail
 
         routing:
-            Symfony\Component\Mailer\Messenger\SendEmailMessage: sync
-            Symfony\Component\Notifier\Message\ChatMessage: async
-            Symfony\Component\Notifier\Message\SmsMessage: async
\ No newline at end of file
+            App\Message\SmsNotification: sms_async
+            Symfony\Component\Mailer\Messenger\SendEmailMessage: mail_async
```

Comment:
I assume you have the infinite loop issue. Did you receive an email using the async flow?